packages/pyright-scip/src/config.ts

```diff
@@ -100,7 +103,7 @@ export class ScipPyrightConfig {
             if (configFilePath) {
                 projectRoot = getDirectoryPath(configFilePath);
             } else {
-                this._console.log(`No configuration file found.`);
+                sendStatus(`No configuration file found.`);
```

Comment:
We don't want these messages to clutter the testing output. `sendStatus` respects the `quiet` setting, so switching to `sendStatus` here.